src/app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.cpp

```diff
@@ -2370,6 +2370,8 @@ void CameraAVStreamMgmtServer::HandleCaptureSnapshot(HandlerContext & ctx,
 
     if (image.data.size() > kMaxSnapshotImageSize)
     {
+        ChipLogError(Zcl, "CameraAVStreamMgmt[ep=%d]: Snapshot image file size=%lu", mEndpointId,
```

Comment:
NIT: it would be nice to have a limit in the log message, e.g.: `...: Snapshot image file size exceeded: %lu > %lu", mEndpointId, static_cast<unsigned long>(image.data.size()), kMaxSnapshotImageSize)`